Sources/web3swift/Web3/Web3.swift

```diff
@@ -21,35 +21,35 @@ extension Web3 {
     }
 
     /// Initialized Web3 instance bound to Infura's mainnet provider.
-    public static func InfuraMainnetWeb3(accessToken: String? = nil) async -> Web3 {
-        let infura = await InfuraProvider(Networks.Mainnet, accessToken: accessToken)!
```

Comment:
Force unwrapping a provider must not be done as we can get a random value in `accessToken` and will crash in case of a connection failure.

Reply:
Does it builds this way?)

Reply:
Personally, I like if configurations crash because something is fundamentally wrong. This would always be something you would catch during development, no? Or what would be a reason to do this _on the fly_ running an app?